CoreService/Trustees.psm1

```diff
@@ -90,6 +90,53 @@ function Get-User
 	}
 }
 
+function Get-Group
+{
+    <#
+    .Synopsis
+    Gets information about the a specific Tridion group.
+
+    .Description
+    Gets a group object containing information about the specified group within Tridion.
+
+    .Inputs
+    None.
+
+    .Outputs
+    Returns an object of type [Tridion.ContentManager.CoreService.Client.TrusteeData].
+
+    .Link
+    Get the latest version of this script from the following URL:
+    https://github.com/pkjaer/tridion-powershell-modules
+
+    .Example
+    Get-TridionGroup "WebMaster"
+    Returns information about the group 'WebMaster'.
+	
+	.Example
+    Get-TridionUser "tcm:2-12-65552"
+    Returns information about group with the id 'tcm:2-12-65552'.
+    
+    #>
+    [CmdletBinding()]
+    Param
+    (
+        [Parameter(ValueFromPipelineByPropertyName=$true)]
+		[ValidateNotNullOrEmpty()]
+        [string]$Group
+    )
+
+	Write-Verbose "Loading Tridion group...";
+	$GroupObj = Get-Groups | ?{($group -eq ($_.Id)) -or ($group -eq $_.Title)} | Select -First 1
```

Comment:
Got you. I'll fix these in the following day and will make another push
request.

Thanks
On Mar 21, 2016 11:31 AM, "Peter Kjaer" <notifications@github.com> wrote:

> In CoreService/Trustees.psm1
> <https://github.com/pkjaer/tridion-powershell-modules/pull/15#discussion_r56796710>
> :
>
> > +	
> > +	.Example
> > +    Get-TridionUser "tcm:2-12-65552"
> > +    Returns information about group with the id 'tcm:2-12-65552'.
> > +
> > +    #>
> > +    [CmdletBinding()]
> > +    Param
> > +    (
> > +        [Parameter(ValueFromPipelineByPropertyName=$true)]
> > +		[ValidateNotNullOrEmpty()]
> > +        [string]$Group
> > +    )
> > +
> > +	Write-Verbose "Loading Tridion group...";
> > +	$GroupObj = Get-Groups | ?{($group -eq ($_.Id)) -or ($group -eq $_.Title)} | Select -First 1
>
> This works, obviously, but it would be more efficient to load the group
> directly if a TCM URI is given. It only needs to load all of the groups if
> a title is given. Perhaps a parameter set is in order?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pkjaer/tridion-powershell-modules/pull/15/files/6df18363718fe61a1c81cd454baa3ecb66312d51#r56796710>
>